pkgs/applications/misc/ianny/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, dbus
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ianny";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "zefr0x";
+    repo = "ianny";
+    rev = "c0e519b3258e837184e639eb9174394be700da7f";
+    hash = "sha256-+z8WXk8x/LHADzhOH6JlSuKDRhNvwrUTyTGcIfHB82k=";
+  };
+
+  cargoHash = "sha256-ieUCWgRumwAW6va8J7OxlQYvQsqGqxYoS9XHpXx++2A=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ dbus.dev ];
```

Comment:
Isn't this picked up by default?

Reply:
No, after testing I can confirm that the build fails after being unable to find pkg-config (if I remove `buildInputs = [ dbus.dev ];`) and it fails after being unable to find dbus-devel (if `buildInputs = [ dbus.dev ];` is removed)